src/diet.ts

```diff
@@ -362,21 +361,29 @@ function copiers(): MenuItem<Note>[] {
 function countCopies(diet: Diet<Note>): number {
   // this only counts the copies not yet realized
   // any copies already realized will be properly counted by embezzlerCount
-  const alreadyGregarious =
-    get("beGregariousCharges") > 0 ||
-    (get("beGregariousFightsLeft") > 0 &&
-      get("beGregariousMonster") === $monster`Knob Goblin Embezzler`);
+  const currentlygregarious =
+    get("beGregariousFightsLeft") > 0 &&
+    get("beGregariousMonster") === $monster`Knob Goblin Embezzler`
+      ? 1
+      : 0;
+  const gregariousCharges = get("beGregariousCharges") ?? 0 + currentlygregarious;
```

Comment:
```suggestion
  const gregariousCharges = (get("beGregariousCharges") ?? 0) + currentlygregarious;
```
also I don't think you should need ??, but maybe you do idk

Reply:
oh i could default get 0 instead

Reply:
i guess what i'm saying is "does that return null sometimes" because if so, that's weird

Reply:
keep using the ?? if you need it, but I should put in a tiny mafia PR to make this default to zero